media_driver/linux/common/ddi/media_libva.cpp

```diff
@@ -5334,12 +5334,21 @@ DdiMedia_QueryVideoProcPipelineCaps(
     }
```

Comment:
Question to the above if:
`if ((context & DDI_MEDIA_MASK_VACONTEXT_TYPE) == DDI_MEDIA_VACONTEXTID_OFFSET_DECODER) {}`
I wonder whether this if can be hit on BDW and what it will actually report? There is no condition like if isSFC()... I wonder whether VACONTEXTID_OFFSET_DECODER assumes SFC or it may work with the AVS as well? If AVS possible, then a lot of SFC defines look suspicious. If only SFC is possible, I suggest to comment this explicitly.

Reply:
This if will be hit when the case is decode+SFC, it will only go through SFC path, so here the restriction here is the max resolution of SFC. I'll comment it here to make it more explicitly, thanks.